api/swagger.yaml

```diff
@@ -1786,7 +1791,7 @@ definitions:
         example: "arm"
       Variant:
         description: |
-          CPU architecture variant (presently ARM-only).
+          CPU architecture variant (ARM only).
```

Comment:
this one's a bit weird too

Reply:
These are the equivalent of the OCI spec fields, so we could look at borrowing descriptions from there (if they're good). I think the "ARM only" still applies, because there's also limited use for some Intel sub-arches.

Reply:
https://github.com/opencontainers/image-spec/blob/main/config.md

<img width="1059" alt="Screenshot 2024-01-10 at 23 00 05" src="https://github.com/moby/moby/assets/1804568/d7216bfd-4b61-4f5b-ae71-4879b3ce4921">

Reply:
OK, let's stick with the existing description for now then, we can revisit to improve descriptions in the future